config/config_pose.yaml

```diff
@@ -32,18 +44,20 @@ draw_colors: {
     "soup": [255, 101, 0],  # orange
     "sugar": [232, 222, 12],  # yellow
     "bleach": [232, 222, 12],  # yellow
-        
 }
 
-# Camera intrinsics (Logitech C920)
-camera_settings: {
-    "name": "logitech_c920",
-    "width": 640,
-    "height": 480,
-    "fx": 641.5,
-    "fy": 641.5,
-    "cx": 320.0,
-    "cy": 240.0
+# optional: provide a transform that is applied to the pose returned by DOPE
+model_transforms: {
+#    "cracker": [[ 0,  0,  1,  0],
+#                [ 0, -1,  0,  0],
+#                [ 1,  0,  0,  0],
+#                [ 0,  0,  0,  1]]
+}
+
+# optional: if you provide a mesh of the object here, a mesh marker will be
+# published for visualization in RViz
+meshes: {
+#  "cracker": "package://dope/meshes/003_cracker_box/google_16k/textured.obj",
```

Comment:
Thank you this is great feature :P

Reply:
Well, thank you for making DOPE open source. This is my way of giving back. :)

Reply:
The DOPE run based on aligned YCB models, generate by: https://github.com/NVIDIA/Dataset_Utilities#nvdu_ycb. Did it work for you when you use the original model?

Reply:
No, I used the aligned models, but I rescaled them to meters. I'll update the comments in `config_pose.yaml` and also add a new parameter `mesh_scales` so that it works with the aligned_cm YCB meshes.

Reply:
Done: https://github.com/NVlabs/Deep_Object_Pose/pull/49/commits/b6b8058d1b9426dd1e68e510ca4e3ab2c44bfb25